upup/pkg/fi/http.go

```diff
@@ -105,6 +159,9 @@ func downloadURLAlways(url string, destPath string, dirMode os.FileMode) error {
 	if err != nil {
 		return fmt.Errorf("Cannot create request: %v", err)
 	}
+	if err := addAuthHeadersFromEnvironment(httpClient, req); err != nil {
+		return fmt.Errorf("Cannot determine authentication headers: %v", err)
```

Comment:
Super-nit: we're trying to use %w for all new error messages (at some stage we'll go back and fix all the existing %v)

Reply:
Cool - did that for this file!